suite-native/app/src/navigation/AppTabNavigator.tsx

```diff
@@ -18,18 +18,15 @@ export const AppTabNavigator = () => (
             initialRouteName={AppTabsRoutes.HomeStack}
             screenOptions={{
                 headerShown: false,
-                unmountOnBlur: false,
+                unmountOnBlur: true,
```

Comment:
Why?

Reply:
It was requested by product to always reset tabs on tab change. That's why there were errors on settings, receive screen and other stacks. I tested that this didn't bring any real performance problems